test/Scripts.Tests/package-release.zip.snapshot

```diff
@@ -40,8 +40,427 @@ Plugins/iOS/SentryNativeBridge.m.meta
 Plugins/iOS/SentryNativeBridgeNoOp.m
 Plugins/iOS/SentryNativeBridgeNoOp.m.meta
 Plugins/iOS/Sentry.xcframework~/ios-arm64/
+Plugins/iOS/Sentry.xcframework~/ios-arm64_x86_64-maccatalyst/
 Plugins/iOS/Sentry.xcframework~/ios-arm64_x86_64-simulator/
+Plugins/iOS/Sentry.xcframework~/macos-arm64_x86_64/
+Plugins/iOS/Sentry.xcframework~/tvos-arm64/
+Plugins/iOS/Sentry.xcframework~/tvos-arm64_x86_64-simulator/
+Plugins/iOS/Sentry.xcframework~/watchos-arm64_arm64_32_armv7k/
+Plugins/iOS/Sentry.xcframework~/watchos-arm64_i386_x86_64-simulator/
+Plugins/iOS/Sentry.xcframework~/xros-arm64/
+Plugins/iOS/Sentry.xcframework~/xros-arm64_x86_64-simulator/
 Plugins/iOS/Sentry.xcframework~/Info.plist
+Plugins/iOS/Sentry.xcframework~/tvos-arm64/dSYMs/
```

Comment:
does it make sense to ship tvos and watchos?

Reply:
and xros, whatever that is

Reply:
> xros

visionOS

Reply:
Unity has these as build targets. We don't specifically have support for those (yet) but it doesn't make sense to me to fix (trim) up the `.xcframework`.
<img width="266" alt="Screenshot 2024-10-21 at 10 27 13" src="https://github.com/user-attachments/assets/b8ff756f-d086-4b20-9440-d379911b08cc">